code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1305,6 +1305,22 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 			var/turf/startT = spaceDebrisStartLoc(startside, T.z)
 			var/turf/endT = spaceDebrisFinishLoc(startside, T.z)
 			new /obj/effect/immovablerod(startT, endT,target)
+		if(ADMIN_PUNISHMENT_SUPPLYPOD)
+			///////load the supply pod up with something!
+			var/target_path = input(usr,"Enter typepath of an atom you'd like to send with the pod (type \"cancel\" to abort smiting):" ,"Typepath","/obj/item/reagent_containers/food/snacks/grown/harebell")

```

Comment:
You can follow up this `input` call with ` as null|text` to give it a cancel button, then check `isnull(target_path)` rather than `target_path == "cancel"`.

Reply:
do isnull() and (!target_path) achieve the same result?